{"target": "tests/test_evaluation.py::TestPipeline::test_pipeline_raise_error_on_undeclared_dataframe", "type": "victim", "patch": false, "polluter": {"tests/test_evaluation.py::TestPipeline::test_pipeline_execute_stmts": [], "tests/test_evaluation.py::TestPipeline::test_pipeline_execute_multiple_stmts": [], "tests/test_evaluation.py::TestMultipleStmts::test_multiple_stmts_with_pipeline": [], "tests/test_evaluation.py::TestAttrCall::test_attr_call_on_var": []}, "time": 320.16436767578125}